plugins/occt/CMakeLists.txt

```diff
@@ -17,7 +17,13 @@ if("${OpenCASCADE_VERSION}" VERSION_LESS "7.5.0")
   message(FATAL_ERROR "Plugin: OpenCASCADE: ${OpenCASCADE_VERSION} is not supported by F3D, please update your OpenCASCADE installation.")
 endif()
 
-if((NOT TARGET "TKSTEP") AND (NOT TARGET "TKIGES") AND (NOT TARGET "TKMesh"))
+if("${OpenCASCADE_VERSION}" VERSION_LESS "7.8.0")
+  if((NOT TARGET "TKSTEP") AND (NOT TARGET "TKIGES") AND (NOT TARGET "TKMesh"))
+    message(FATAL_ERROR "Plugin: OpenCASCADE does not contain required modules")
+  endif()
+endif()
+
+if((NOT TARGET "TKDESTEP") AND (NOT TARGET "TKDEIGES") AND (NOT TARGET "TKMesh"))
```

Comment:
so TKDESTEP is a new module or a module that is shared between all supported versions of OCCT ?

Reply:
looks like it `TKSTSEP` and `TKIGES` have been renamed to `TKDESTEP` and `TKDEIGES` according to the initial diff from #1304 
```diff
-if((NOT TARGET "TKSTEP") AND (NOT TARGET "TKIGES") AND (NOT TARGET "TKMesh"))
+if((NOT TARGET "TKDESTEP") AND (NOT TARGET "TKDEIGES") AND (NOT TARGET "TKMesh"))
```

Reply:
> looks like it `TKSTSEP` and `TKIGES` have been renamed to `TKDESTEP` and `TKDEIGES` according to the initial diff from #1304

Yes exactly: the rule is now:

DE TK components have been combined or separated based on specific CAD formats to support plug-in ability.

    Components now have a "TKDE" prefix. The available list includes TKDESTEP, TKDEOBJ, TKDEIGES, TKDEGLTF, TKDEVRML, TKDEPLY, TKDESTL.
    The DE DRAW TK has been updated in a similar way: DRAW components now have a "TKXSDRAW" prefix. The available list includes TKXSDRAWSTEP, TKXSDRAWOBJ, TKXSDRAWIGES, TKXSDRAWGLTF, TKXSDRAWVRML, TKXSDRAWPLY, TKXSDRAWSTL.